workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -437,6 +449,48 @@ for (name in names(ll)){
 }
 ```
 
+```{r groupcounts, cache=TRUE}
+counts.list <- list()
+
+# compute aggregate (by gene) normalized transcript counts from Salmon
+transcript.counts <- aggregate(transcript.counts, list(transcript.geneids), sum)
+
+# Use normalized counts to get per-level mean counts. Attach Salmon aggregate
+# counts to each output file. This does ~not~ assume any particular model.
+for (name in names(res.list)) {
+    counts.list[[name]] <- res.list[[name]][,c(1:4,7)]
```

Comment:
would it be safer to use named columns rather than `c(1:4, 7)`? I don't know what these columns are off the top of my head.

Reply:
yeah so it's grabbing gene, base mean, log2 fold change, stderr, and padj in this situation. It might be safer, it might not. It depends on whether you think it's more likely that the DESeq2 results format itself changes or the colnames change. I think the rest of the file uses colnames? So I can change it to use those as well.

Reply:
Changed to use colnames